### ISSUE TITLE
"killOthers" missing from "ConcurrentlyOptions" type

### ISSUE THREAD

**Issue by electrovir** [open-cli-tools/concurrently#424]
`killOthers` is listed in the README: https://github.com/open-cli-tools/concurrently/blob/a0dcbd9bf2917401d777c567227a4aeabe3d17ba/README.md?plain=1#L330-L331

But it is not listed in the types: https://github.com/open-cli-tools/concurrently/blob/a0dcbd9bf2917401d777c567227a4aeabe3d17ba/src/concurrently.ts#L51-L125

**Comment by electrovir** [comment]
It appears that my confusion stems from multiple duplicate variable names within the code.

_This_ copy of `ConcurrentlyOptions` does indeed contain `killOthers`: https://github.com/open-cli-tools/concurrently/blob/a0dcbd9bf2917401d777c567227a4aeabe3d17ba/src/index.ts#L21-L92

In order to use _that_ copy of `ConcurrentlyOptions`, the default export from `concurrently` must be used, rather than the destructed export. Note that my `concurrently` api command was also functioning entirely incorrectly with a destructured import; not seeing the `killOthers` type was merely a correlated issue.


**tl;dr**
This works:
```typescript
import concurrently from 'concurrently';
```
This does not work (it runs but it bypasses a lot of code that is necessary for expected operation according to the docs):
```typescript
import {concurrently} from 'concurrently';
```

**Comment by paescuj** [comment]
I think that's what we track in #399.
